pandas/tests/series/methods/test_drop_duplicates.py

```diff
@@ -249,3 +249,10 @@ def test_drop_duplicates_ignore_index(self):
         result = ser.drop_duplicates(ignore_index=True)
         expected = Series([1, 2, 3])
         tm.assert_series_equal(result, expected)
+
+    def test_duplicated_arrow_dtype(keep):
```

Comment:
Did you intend to parameterize over `keep`?

Reply:
Nope, that was a lesson that you shouldn't copy paste without double checking